docs/resources/service_integration_endpoint.md

```diff
@@ -137,17 +137,17 @@ Required:
 Required:
 
 - `bootstrap_servers` (String) Bootstrap servers.
-- `security_protocol` (String) Security protocol.
+- `security_protocol` (String) Enum: `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT`, `SASL_SSL`. Security protocol.
 
 Optional:
 
-- `sasl_mechanism` (String) SASL mechanism used for connections to the Kafka server.
+- `sasl_mechanism` (String) Enum: `PLAIN`, `SCRAM-SHA-256`, `SCRAM-SHA-512`. SASL mechanism used for connections to the Kafka server.
 - `sasl_plain_password` (String, Sensitive) Password for SASL PLAIN mechanism in the Kafka server.
 - `sasl_plain_username` (String) Username for SASL PLAIN mechanism in the Kafka server.
 - `ssl_ca_cert` (String) PEM-encoded CA certificate.
 - `ssl_client_cert` (String) PEM-encoded client certificate.
 - `ssl_client_key` (String) PEM-encoded client key.
-- `ssl_endpoint_identification_algorithm` (String) The endpoint identification algorithm to validate server hostname using server certificate.
+- `ssl_endpoint_identification_algorithm` (String) Enum: `https`, ``. The endpoint identification algorithm to validate server hostname using server certificate.
```

Comment:
🔎 This should be fixed in go-api-schemas https://github.com/aiven/go-api-schemas/blob/25035ec40a7db6153c6e992f86765de7a8064a0d/pkg/dist/integration_endpoint_types.yml#L311. Can be done later.